### PR TITLE
feat(pci.ai): datatr-180 - ai deploy - add label free during beta

### PR DESCRIPTION
**Pull request by jperchoc**
datatr-180

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #datatr-180
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Add a label "free during beta" for ai deploy

### :flags: Translations

- [x] TRANS-49272

### CODE REVIEW

**Review by ClaveauAlex — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-review/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "pci_app_add_review_scaling_trigger_resource_threshold": "Seuil de déclenchement :",
   "pci_app_add_review_price_total": "Prix total",
   "pci_app_add_review_price_info": "{{ price }} HT /heure",
+  "pci_app_add_review_price_free_during_beta": "Gratuit pendant la beta",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-review/translations/Messages_fr_FR.json:18]:
```suggestion
  "pci_app_add_review_price_free_during_beta": "Gratuit pendant la bêta",
```

**Review by mohammed-zahaf — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-review/translations/Messages_fr_FR.json (hunk 1 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-review/translations/Messages_fr_FR.json]: Changed here: https://github.com/ovh/manager/pull/8802/commits/638f1cf72e1f7d54052fb7fdf18618dd533f91fd
Thanks !

**Review by mohammed-zahaf — APPROVED** [review]

**Review by y4nnL — COMMENTED** [review]
Hello,
Have you considered adding a `beta` option to the `ovh-manager-catalog-price` component that defaults to `false` to prevent any side effects, and would display a beta badge and `<del></del>` its content when it is set to `true` ?

**Review by ClaveauAlex — APPROVED** [review]

**Review by y4nnL — CHANGES_REQUESTED** [review]
@jperchoc can you please have a look at my previous comment
Other projects could use this option :+1:

**Comment by jperchoc** [comment]
Hi @y4nnL , this sound like an awesome feature that could/should be added to the manager project. It would also simplify the tags on the left menu, which are hardcoded too.
However, I'm afraid that this is more a feature request, that should be handled in a separate PR, with further testing and QA, as it could break existing things. 
This PR is expected to reach prod quickly, as it is a very important information for our users.

File: packages/manager/modules/catalog-price/src/catalog-price.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <span data-ng-if="!$ctrl.fromToPrice">
     <span data-ng-if="::$ctrl.price || $ctrl.showZeroPrice">
         <span data-ng-if="::$ctrl.isAsiaFormat()">
-            <strong data-ng-class="{ 'd-block': $ctrl.block }">
+            <strong data-ng-class="{ 'd-block': $ctrl.block}">
```
Comment (ClaveauAlex) [inline packages/manager/modules/catalog-price/src/catalog-price.html:4]:
```suggestion
            <strong data-ng-class="{ 'd-block': $ctrl.block }">
```

File: packages/manager/modules/catalog-price/src/catalog-price.controller.js [hunk 3]
```diff
@@ -20,6 +20,9 @@ export default class {
   }
 
   $onInit() {
+    // Set bate to false by default
```
[the remark this answers is not in the document]
Comment (jperchoc) [reply packages/manager/modules/catalog-price/src/catalog-price.controller.js]: fixed: https://github.com/ovh/manager/pull/8802/commits/fd17b19a828fc8baec980f1b8951b7f43d8a7a75

File: packages/manager/modules/catalog-price/src/catalog-price.html (hunk 2 above)
Comment (jperchoc) [reply packages/manager/modules/catalog-price/src/catalog-price.html]: Fixed: https://github.com/ovh/manager/pull/8802/commits/fd17b19a828fc8baec980f1b8951b7f43d8a7a75

**Review by ClaveauAlex — DISMISSED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by ClaveauAlex — APPROVED** [review]

**Review by y4nnL — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**